public/docs/_examples/toh-6/ts/app/app.module.ts

```diff
@@ -12,7 +12,7 @@ import { HttpModule }    from '@angular/http';
 
 // #enddocregion v1
 // Imports for loading & configuring the in-memory web api
-import { InMemoryWebApiModule } from 'angular-in-memory-web-api';
+import { InMemoryWebApiModule } from 'angular-in-memory-web-api/in-memory-web-api.module';
```

Comment:
I wonder if this is happening because the code is already converted to JavaScript by the time it enters the rollup world. I guess the barrel is transpiled as well, but maybe Rollup isn't considering the convention of using barrels...